auto_nag/scripts/regression.py

```diff
@@ -5,33 +5,22 @@
 from libmozdata.bugzilla import Bugzilla
 import re
 from auto_nag.bzcleaner import BzCleaner
-
-
-COMMENTS_PAT = re.compile('^>.*[\n]?', re.MULTILINE)
-HAS_UPLIFT_PAT = re.compile(
-    r'(Feature/Bug causing the regression)|(feature/regressing bug #)', re.I
-)
-UPLIFT1_PAT = re.compile(
-    r'[\[]?Feature/Bug causing the regression[\]]?:\n*(?:(?:[ \t]*)|(?:[^0-9]*bug[ \t]*))([0-9]+)[^\n]*$',
-    re.MULTILINE | re.I,
-)
-UPLIFT2_PAT = re.compile(
-    r'[\[]?Bug caused by[\]]? \(feature/regressing bug #\):\n*(?:(?:[ \t]*)|(?:[^0-9]*bug[ \t]*))([0-9]+)[^\n]*$',
-    re.MULTILINE | re.I,
-)
-REG_BY_BUG_PAT = re.compile(
-    r'[ \t]regress[^0-9\.,;\n]+(?:bug[ \t]*)([0-9]+)(?:[^\.\n\?]*[\.\n])?', re.I
-)
-CAUSED_BY_PAT = re.compile('caused by bug[ \t]*([0-9]+)', re.I)
-REG_PAT = re.compile(
-    r'(regression is)|(regression range)|(regressed build)|(mozregression)|(this is a regression)|(this regression)|(is a recent regression)|(regression version is)|(regression[- ]+window)',
-    re.I,
-)
+try:
+    import lzma
+except ImportError:
+    from backports import lzma
+import shutil
+from bugbug.models.regression import RegressionModel
 
 
 class Regression(BzCleaner):
     def __init__(self):
```

Comment:
How often is this executed? On every autonag run? Or does it run like a daemon?
If it is executed every time, I'll have to add some code to only download the model when necessary.

Reply:
Once a day with a cron job

Reply:
We could run the jobs with autofix more often. Besides the load on bz, we could do it!

Reply:
OK, I'll do a `head` request and store the Etag, so we only re-download the model when it changes.